components/Wallet/Wallet.tsx

```diff
@@ -29,10 +29,10 @@ export function Wallet() {
   useEffect(() => {
     if (!connect) return;
 
-    const previousConnectedWallets = JSON.parse(window.localStorage.getItem("connectedWallets") || "[]");
+    const previousConnectedWallets = JSON.parse(window.localStorage.getItem("connectedWallets") || "[]") as string[];
 
     if (previousConnectedWallets?.length) {
-      (async () => {
+      void (async () => {
```

Comment:
New plugin suggests using `void` here to indicate that this function is not actually meant to return a promise even though it's marked as `async`. We get better typings like this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void

Reply:
interesting, never seen this